mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -4116,6 +4148,33 @@ LogicalResult LLVM::masked_scatter::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// masked_expandload (intrinsic)
+//===----------------------------------------------------------------------===//
+
+void LLVM::masked_expandload::build(OpBuilder &builder, OperationState &state,
+                                    mlir::TypeRange resTys, Value ptr,
+                                    Value mask, Value passthru,
+                                    uint64_t align) {
+  ArrayAttr callArgs = getLLVMAlignParamForCompressExpand(builder, true, align);
```

Comment:
```suggestion
  ArrayAttr argAttrs = getLLVMAlignParamForCompressExpand(builder, true, align);
```
nit: I would use argAttrs here and for the other intrinsic since it is not really a call in LLVM dialect.